client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SecureExecTest.java

```diff
@@ -12,131 +22,163 @@
 import org.infinispan.security.AuthorizationPermission;
 import org.infinispan.security.Security;
 import org.infinispan.security.impl.IdentityRoleMapper;
+import org.infinispan.server.core.security.simple.SimpleServerAuthenticationProvider;
 import org.infinispan.server.hotrod.test.TestCallbackHandler;
 import org.infinispan.test.TestingUtil;
 import org.infinispan.test.fwk.CleanupAfterMethod;
 import org.infinispan.test.fwk.TestCacheManagerFactory;
 import org.testng.annotations.Test;
 
-import javax.security.auth.Subject;
```

Comment:
Actually, leave it as it is because this is already in JDG :( - Any more changes here will create further problems :(